betty/cropper/templates/image.js

```diff
@@ -33,6 +33,19 @@
     };
   }
 
+  // Credit to http://ejohn.org/blog/getboundingclientrect-is-awesome/
+  function isInViewport(el) {
```

Comment:
Hm, there's not a simple way to get that function in to this js file. Probably a low ROI for re-use.